coorx/systems.py

```diff
@@ -121,24 +123,55 @@ def transform(self, cs1:CoordSysOrStr, cs2:CoordSysOrStr):
 
         raise TypeError(f"No transform defined linking '{cs1}' to '{cs2}'")
 
+    def transform_path(self, start, end) -> list:
+        """Return a list of transforms needed to map from start to end."""
+        start, end = self.system(start), self.system(end)
+        if start == end:
+            return []
+        found = self._find_path(start, end, set())
+        if found:
+            return [start] + found
+        return []
```

Comment:
Raise exception instead?

Reply:
I guess that's not true. My attraction to exceptions is fickle, though, and I don't know why I prefer them sometimes and not others.